pandas/core/strings.py

```diff
@@ -2094,8 +2094,46 @@ def encode(self, encoding, errors="strict"):
         return self._wrap_result(result)
 
     _shared_docs['str_strip'] = ("""
-    Strip whitespace (including newlines) from each string in the
-    Series/Index from %(side)s. Equivalent to :meth:`str.%(method)s`.
+    Strip whitespaces from string in Series.
+
+    Strip whitespace (including newlines) or given string from
+    each string in the Series/Index from %(side)s. Equivalent to
+    :meth:`str.%(method)s`.
+
+    Parameters
+    ----------
+    to_strip : str or unicode
+        String or unicode to strip in the given string.
+
+    Examples
+    --------
+    >>> # strip method
+    >>> s = pd.Series(['   This is a Test 1  '])
```

Comment:
I'd create a single Series with all the values you want to use in the examples, and then apply all operations to it.